src/main/views/subscription-urn-search-results.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "./macros/common-components.njk" import submitButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block pageTitle %}
+  {{ "Subscription URN Search Results" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();event.preventDefault();"
+    }
+  }) }}
```

Comment:
When I search for a URN, it showed me the result page. But on this page, when I click on Back button, it showed an error. It is happening on Chrome, but on Safari, it is working fine.

Reply:
I can't replicate this. I can click on the back button and it is working correctly in Chrome, Safari and Firefox